app/models/MetricsData.scala

```diff
@@ -0,0 +1,31 @@
+package models
+
+sealed trait PublishingSystem
+case object Composer extends PublishingSystem
+case object InCopy extends PublishingSystem
+
+case class Metrics (
+    id: String,
+    contentCreated: Long,
+    composerId: Option[String],
+    storyBundleId: Option[String],
+    firstSeenInWorkflow: Option[Long],
```

Comment:
This may be a silly question, but is there value in using something like [Joda](http://www.joda.org/joda-time/) for these date/time types? I'm not sure of the advantages...